### ISSUE TITLE
There is Milvus artifact but not documentation

### ISSUE THREAD

**Issue by lordofthejars** [quarkiverse/quarkus-langchain4j#1136]
Is there any reason to have the Quarkus Milvus integration but not being documented?

**Comment by jmartisk** [comment]
Yeah I could come up with some excuses like "I haven't had time", or "Nobody is using it anyway" or so. I don't know if anybody is using it  with Quarkus, I haven't seen anybody. It just thinly wraps the upstream langchain4j implementation, so anything written in https://docs.langchain4j.dev/integrations/embedding-stores/milvus should be still applicable, we just add a configuration model on top. So yeah, we could certainly at least add a configuration reference page.

**Comment by lordofthejars** [comment]
Well, I am developing some demos, and I use PGVector and Chroma. Now, I want to try Milvus just to have all of them

**Comment by lordofthejars** [comment]
Sure. Tomorrow I will start playing with it.

**Comment by lordofthejars** [comment]
If you want, leave the issue open for some time, so if I find anything, I can add it here like the `dimension` property.

BTW, couldn't it be injected by calling the `embedding.dimension()` function automatically?

**Comment by jmartisk** [comment]
> BTW, couldn't it be injected by calling the `embedding.dimension()` function automatically?

We've discussed this in the past, for in-process embedding models it wouldn't be too hard because the dimension is known, but with remote models, it is hard, because just from a string ID of the model, you can't deduce the dimension without actually calling the model, and the framework probably shouldn't do anything like that by itself.

**Comment by jmartisk** [comment]
Also also we don't know for sure, if there are multiple embedding models in the app (with potentially different dimensions), which one you're going to use with the embedding store.

**Comment by jmartisk** [comment]
fixed by https://github.com/quarkiverse/quarkus-langchain4j/pull/1229
